Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs

```diff
@@ -536,7 +536,7 @@ public async Task UpdatesNameObsolete()
             using (var repoContext = await _github.CreateUserRepositoryContext())
             {
                 var updatedName = Helper.MakeNameWithTimestamp("updated-repo");
-                var update = new RepositoryUpdate(updatedName);
+                var update = new RepositoryUpdate() { Name = updatedName };
```

Comment:
This addressed a compiler warning - I wanted to ensure the output was as clean as possible and, therefore, clean a little bit of the house regardless of whether it was a related change or not.

Reply:
These two tests were there to test the obsolete method, which technically does still work. The previous 2 tests in the file check the newer way of doing it, so if we don't want the warnings it would be better to get rid of those two tests (ending Obsolete) entirely 👍

Reply:
Great point.  I'm thinking we also opt to remove the constructor as well given we are making a breaking change with this change set - and I can note it in the release.